tests/test_preprints.py

```diff
@@ -256,6 +256,13 @@ class TestOnPreprintUpdatedTask(OsfTestCase):
     def setUp(self):
         super(TestOnPreprintUpdatedTask, self).setUp()
         self.user = AuthUserFactory()
+        if len(self.user.fullname.split(' ')) > 2:
```

Comment:
No, but it won't break anything. I wrote this before realizing that the new way of name generation could cause any number of tests to break.